source/voice_control/about_wake_word.markdown

```diff
@@ -78,9 +78,30 @@ How wake words integrate into Home Assistant
 
 As an example, we’re also making the **Porcupine (v1)** wake word engine available. It supports 29 wake words across English, French, Spanish, and German. The wake words include *Computer*, *Framboise*, *Manzana*, and *Stachelschwein*.
 
+## About on-device wake word processing (microWakeWord)
+
+The [microWakeWord](https://github.com/kahrendt/microWakeWord) created by [Kevin Ahrendt](https://www.kevinahrendt.com/) enables ESPHome to detect wake words on devices like the ESP32-S3-BOX-3.
+
+Because openWakeWord is too large to run on low-power devices like the S3-BOX-3, openWakeWord runs wake word detection on the Home Assistant server.
+
+<p><img src='/images/assist/assist_open-vs-micro-wake-word.png' class='no-shadow' /></p>
```

Comment:
I feel like this screenshot doesn't add value here? It was only used to explain why we shipped openWakeWord previously.